vision/google/cloud/proto/vision/v1/text_annotation_pb2.py

```diff
@@ -558,13 +558,14 @@
   DESCRIPTOR = _TEXTANNOTATION,
   __module__ = 'google.cloud.proto.vision.v1.text_annotation_pb2'
   ,
-  __doc__ = """TextAnnotation contains a structured representation of OCR extracted
-  text. The hierarchy of an OCR extracted text structure is like this:
-  TextAnnotation -> Page -> Block -> Paragraph -> Word -> Symbol Each
+  __doc__ = """``TextAnnotation`` contains a structured representation of OCR extracted text.
```

Comment:
That would mean that it's outputting bad rst (see [here](https://www.mantidproject.org/Documentation_Guide_For_Devs#Inline_interpreted_text_or_phrase_reference_start-string_without_end-string)). 

I wonder if you would be willing to add both of these cases you ran into into the test case for sphinx-docstring-typing so we can see exactly what's going on.